### ISSUE TITLE
Feature request: Subscribe to keyword

### ISSUE THREAD

**Issue by lig2017** [dandi/dandi-archive#1974]
Hi, 
Thanks for doing this :)
I am from the NeuroReHack_2024 and we came up with this idea. It would be interesting to have a subscription to DANDI Archive so that whenever a new Dandiset is uploaded related to a keyword I subscribed to, I would get an email about it.  

Thank you!

**Comment by waxlamp** [comment]
Good idea. We will look into it. Thanks for the feature request, @lig2017!
